src/services/home-dashboard/modules/PersonalHealthDashboard.vue

```diff
@@ -304,4 +323,8 @@ export default {
         }
     }
 }
+:deep(.p-empty) {
```

Comment:
nitpick: please add comment when you custom mirinae component!

```suggestion
}
/* custom design-system component - p-empty */
:deep(.p-empty) {
```